graphql/resolvers/users.js

```diff
@@ -1089,5 +1099,25 @@ module.exports = {
       });
       return users;
     },
+
+    async insertPersEmailProp() {
+      var users = await User.find();
+      users.forEach(async function(user){
+        const email = user.email;
+        if(!user.personalEmail){
+          const updatedUser = await User.findOneAndUpdate(
```

Comment:
we already queried for the user (its our user object here).
```suggestion
          user.personalEmail = "";
          user.save((err, updatedUser) => {
          if (err) {
            // Handle error
          } else {
            console.log('Document updated:', updatedUser);
          }
```